hedera-node/src/main/resources/bootstrap.properties

```diff
@@ -149,6 +149,7 @@ hedera.recordStream.logPeriod=2
 hedera.recordStream.queueCapacity=5000
 hedera.recordStream.recordFileVersion=5
 hedera.recordStream.signatureFileVersion=5
+hedera.recordStream.enableTraceabilityMigration=false
```

Comment:
Why are we defaulting to false?

Reply:
I'm not sure if we reached an agreement the last time we discussed the issue regarding which environments will perform the migration (@stoqnkpL @tinker-michaelj), so it was left here to `false` with the possibility of being overridden by the properties files of the different environments if needed